app/components/dashboard/show/works_list_component.rb

```diff
@@ -4,14 +4,33 @@ module Dashboard
   module Show
     # Component for rendering a table on the work show page.
     class WorksListComponent < ApplicationComponent
-      def initialize(collection:)
+      def initialize(collection:, status_map:)
         @collection = collection
+        @status_map = status_map
         super()
       end
 
       attr_reader :collection
 
       delegate :works, to: :collection
+
+      def id
+        @id ||= dom_id(collection, 'table')
+      end
+
+      def id_for(work)
+        dom_id(work, id)
+      end
+
+      def values_for(work)
+        [
+          link_to(work.title, work_path(work)),
+          @status_map[work.id].status_message,
```

Comment:
I'm fuzzy on this so take it with a grain of salt: could we use ViewComponent's "collections" (or slots) handle these values? Each would be its own component, I guess. I'm still trying to get a handle on what the sweet spot is for structuring our components.

Reply:
Currently we are providing a hash of values for each row to a slot which is used to construct a TableRowComponent.

Alternatively, we could provide an array of hashes of values that would be used by to construct TableRowComponents. This is show here: https://viewcomponent.org/guide/slots.html#rendering-collections

I would consider this a slight stylistic difference, but not really substantive. (I have a minor preference for the way it is in this component because providing an array of hashes seems sorta non-intuitive.)